src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -746,6 +746,76 @@ def iot_hub_manual_failover(cmd, client, hub_name, failover_region, resource_gro
     return iot_hub_get(client, hub_name, resource_group_name)
 
 
+def digitaltwin_create_repository(cmd, client, repo_endpoint, repo_name):
+    return _digitaltwin_create_update_repository(cmd, client, repo_endpoint, repo_name)
+
+
+def digitaltwin_update_repository(cmd, client, repo_endpoint, repo_id, repo_name):
+    return _digitaltwin_create_update_repository(cmd, client, repo_endpoint, repo_name, repo_id)
+
+
+def digitaltwin_list_repository(cmd, client, repo_endpoint):
+    headers = {'Authorization': '{}'.format(generate_bearer_token(cmd))}
```

Comment:
given this code is repeated and has the same pattern consider extracting this into a function?
```suggestion
    headers = _get_auth_header(cmd)
```

```
def _get_auth_header(cmd):
    return {'Authorization': '{}'.format(generate_bearer_token(cmd))}
```

@anusapan

Reply:
Done